browser/main/lib/AwsMobileAnalyticsConfig.js

```diff
@@ -18,9 +18,10 @@ function initAwsMobileAnalytics () {
   AWS.config.credentials.get((err) => {
     if (!err) {
       console.log('Cognito Identity ID: ' + AWS.config.credentials.identityId)
+      recordDynamicCustomEvent('MAIN_FOCUSED')
```

Comment:
Is this really `MAIN_FOCUSED`?  In my thought, `MAIN_FOCUSED` is this action (gif)
![60e2b1f329f82d2472beb68bf7bfea95](https://user-images.githubusercontent.com/11307908/30355767-54c09b9e-9870-11e7-813c-962b0b5b2b3d.gif)

What do you suppose to `MAIN_FOCUSED`?

Reply:
Well, it certainly is.
Since mixpanel treats such an event as 'MAIN_FOCUSED', I thought that it is easy for the founder to understand, so I made this event name like this.
But this is hard to understand for user, so I will change it to 'START'.
Or maybe you come up with something good name?

Reply:
I understand your thoughts. `APP_STARTED` or `LAUNCHED` is good for it because if `START`, I might think 'start what?' (`LAUNCH` itself indicates that an application starts).

Reply:
thank you for your opinion.
Base your opinion, I think that this event's name is  'APP_STARTED' is befitting.